### ISSUE TITLE
Missing Alt Text

### ISSUE THREAD

**Issue by maschad96** [MicrosoftDocs/iis-docs#507]
**Description:**
On lines 53, 56, and 68, the images shown are missing alt text.

**File:** 
`iis\application-frameworks\running-classic-asp-applications-on-iis-7-and-iis-8\classic-asp-not-installed-by-default-on-iis.md`

**Page URL:** 
https://docs.microsoft.com/en-us/iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-not-installed-by-default-on-iis
